[code format] gocode check should have rule and use CI to lock down rules

<!-- Please only use this template for submitting feature requests -->

**What would you like to be added**:
Good go code format is the fundamental part of project kruise.

With the tool [gomentalinter](https://github.com/alecthomas/gometalinter)(deprecated, we will use https://github.com/golangci/golangci-lint later), we use the command `gometalinter --disable-all --skip vendor -E gofmt -E goimports -E golint -E goconst -E ineffassign -E misspell -E vet -d ./...` and get the following results:

```
pkg/controller/broadcastjob/broadcastjob_controller_test.go:83:2:warning: ineffectual assignment to err (ineffassign)
pkg/controller/broadcastjob/broadcastjob_controller_test.go:87:2:warning: ineffectual assignment to err (ineffassign)
pkg/controller/broadcastjob/broadcastjob_controller_test.go:128:2:warning: ineffectual assignment to err (ineffassign)
pkg/controller/broadcastjob/broadcastjob_controller_test.go:136:2:warning: ineffectual assignment to err (ineffassign)
pkg/controller/broadcastjob/broadcastjob_controller_test.go:177:2:warning: ineffectual assignment to err (ineffassign)
pkg/controller/broadcastjob/broadcastjob_controller_test.go:236:2:warning: ineffectual assignment to err (ineffassign)
DEBUG: [Jul 11 02:21:06.391] nolint: parsing pkg/controller/statefulset/stateful_set_control_test.go took 8.919083ms
DEBUG: [Jul 11 02:21:06.391] nolint: parsing pkg/controller/statefulset/statefulset_controller_test.go for directives
pkg/controller/statefulset/stateful_set_control_test.go:255:6:warning: ineffectual assignment to pods (ineffassign)
DEBUG: [Jul 11 02:21:06.394] nolint: parsing pkg/controller/statefulset/statefulset_controller_test.go took 3.449273ms
DEBUG: [Jul 11 02:21:06.394] nolint: parsing test/e2e/apps/statefulset.go for directives
pkg/controller/statefulset/statefulset_controller_test.go:119:2:warning: ineffectual assignment to pods (ineffassign)
pkg/controller/statefulset/statefulset_controller_test.go:173:2:warning: ineffectual assignment to pods (ineffassign)
pkg/controller/statefulset/statefulset_controller_test.go:665:3:warning: ineffectual assignment to pod (ineffassign)
DEBUG: [Jul 11 02:21:06.399] nolint: parsing test/e2e/apps/statefulset.go took 4.767768ms
test/e2e/apps/statefulset.go:1264:2:warning: ineffectual assignment to currentRevision (ineffassign)
test/e2e/apps/statefulset.go:1264:19:warning: ineffectual assignment to updateRevision (ineffassign)
DEBUG: [Jul 11 02:21:06.489] [misspell.6]: misspell hits 1: ^(?P<path>.*?\.go):(?P<line>\d+):(?P<col>\d+):\s*(?P<message>.*)$
DEBUG: [Jul 11 02:21:06.489] [misspell.6]: misspell linter took 157.240526ms
DEBUG: [Jul 11 02:21:06.489] nolint: parsing pkg/controller/broadcastjob/broadcastjob_utils.go for directives
DEBUG: [Jul 11 02:21:06.489] nolint: parsing pkg/controller/broadcastjob/broadcastjob_utils.go took 496.757µs
pkg/controller/broadcastjob/broadcastjob_utils.go:43:78:warning: "succceeded" is a misspelling of "succeeded" (misspell)
DEBUG: [Jul 11 02:21:06.501] [gofmt.1]: gofmt hits 3: ^(?P<path>.*?\.go)$
DEBUG: [Jul 11 02:21:06.501] nolint: parsing pkg/controller/broadcastjob/broadcastjob_controller.go for directives
DEBUG: [Jul 11 02:21:06.501] [gofmt.1]: gofmt linter took 173.321387ms
DEBUG: [Jul 11 02:21:06.503] nolint: parsing pkg/controller/broadcastjob/broadcastjob_controller.go took 2.036661ms
DEBUG: [Jul 11 02:21:06.503] nolint: parsing pkg/controller/statefulset/stateful_inplace_utils_test.go for directives
pkg/controller/broadcastjob/broadcastjob_controller.go:1::warning: file is not gofmted with -s (gofmt)
DEBUG: [Jul 11 02:21:06.504] nolint: parsing pkg/controller/statefulset/stateful_inplace_utils_test.go took 856.035µs
DEBUG: [Jul 11 02:21:06.504] nolint: parsing pkg/webhook/default_server/pod/mutating/pod_create_handler.go for directives
pkg/controller/statefulset/stateful_inplace_utils_test.go:1::warning: file is not gofmted with -s (gofmt)
DEBUG: [Jul 11 02:21:06.505] nolint: parsing pkg/webhook/default_server/pod/mutating/pod_create_handler.go took 464.38µs
pkg/webhook/default_server/pod/mutating/pod_create_handler.go:1::warning: file is not gofmted with -s (gofmt)
DEBUG: [Jul 11 02:21:06.782] [golint.3]: golint hits 160: ^(?P<path>.*?\.go):(?P<line>\d+):(?P<col>\d+):\s*(?P<message>.*)$
docs/tutorial/v1/images/guestbook/main.go:37:8:warning: should omit type map[string][]string from declaration of var lists; it will be inferred from the right-hand side (golint)
docs/tutorial/v1/images/guestbook/main.go:40:6:warning: exported type Input should have comment or be unexported (golint)
docs/tutorial/v1/images/guestbook/main.go:44:1:warning: exported function GetList should have comment or be unexported (golint)
docs/tutorial/v1/images/guestbook/main.go:65:1:warning: exported function AppendToList should have comment or be unexported (golint)
docs/tutorial/v1/images/guestbook/main.go:85:1:warning: exported function ListRangeHandler should have comment or be unexported (golint)
docs/tutorial/v1/images/guestbook/main.go:100:1:warning: exported function ListPushHandler should have comment or be unexported (golint)
docs/tutorial/v1/images/guestbook/main.go:119:1:warning: exported function InfoHandler should have comment or be unexported (golint)
docs/tutorial/v1/images/guestbook/main.go:136:1:warning: exported function EnvHandler should have comment or be unexported (golint)
docs/tutorial/v1/images/guestbook/main.go:153:1:warning: exported function HelloHandler should have comment or be unexported (golint)
DEBUG: [Jul 11 02:21:06.782] nolint: parsing hack/gen-openapi-spec/main.go for directives
docs/tutorial/v1/images/guestbook/main.go:165:2:warning: don't use underscores in Go names; var master_port should be masterPort (golint)
DEBUG: [Jul 11 02:21:06.783] nolint: parsing hack/gen-openapi-spec/main.go took 292.268µs
DEBUG: [Jul 11 02:21:06.783] nolint: parsing pkg/apis/apps/v1alpha1/broadcastjob_types.go for directives
hack/gen-openapi-spec/main.go:20:6:warning: func generateSwaggerJson should be generateSwaggerJSON (golint)
DEBUG: [Jul 11 02:21:06.783] nolint: parsing pkg/apis/apps/v1alpha1/broadcastjob_types.go took 340.518µs
DEBUG: [Jul 11 02:21:06.783] nolint: parsing pkg/apis/apps/v1alpha1/defaults.go for directives
pkg/apis/apps/v1alpha1/broadcastjob_types.go:74:6:warning: exported type CompletionPolicyType should have comment or be unexported (golint)
pkg/apis/apps/v1alpha1/broadcastjob_types.go:124:6:warning: exported type JobConditionType should have comment or be unexported (golint)
DEBUG: [Jul 11 02:21:06.783] nolint: parsing pkg/apis/apps/v1alpha1/defaults.go took 85.261µs
DEBUG: [Jul 11 02:21:06.783] nolint: parsing pkg/apis/apps/v1alpha1/sidecarset_types.go for directives
pkg/apis/apps/v1alpha1/defaults.go:26:6:warning: don't use underscores in Go names; func SetDefaults_BroadcastJob should be SetDefaultsBroadcastJob (golint)
DEBUG: [Jul 11 02:21:06.783] nolint: parsing pkg/apis/apps/v1alpha1/sidecarset_types.go took 149.014µs
DEBUG: [Jul 11 02:21:06.783] nolint: parsing pkg/client/generic_client.go for directives
pkg/apis/apps/v1alpha1/sidecarset_types.go:35:6:warning: exported type SidecarContainer should have comment or be unexported (golint)
DEBUG: [Jul 11 02:21:06.783] nolint: parsing pkg/client/generic_client.go took 123.118µs
DEBUG: [Jul 11 02:21:06.783] nolint: parsing pkg/client/registry.go for directives
pkg/client/generic_client.go:9:6:warning: exported type GenericClientset should have comment or be unexported (golint)
DEBUG: [Jul 11 02:21:06.784] nolint: parsing pkg/client/registry.go took 90.133µs
DEBUG: [Jul 11 02:21:06.784] nolint: parsing pkg/controller/statefulset/stateful_inplace_utils.go for directives
pkg/client/registry.go:11:1:warning: exported function NewRegistry should have comment or be unexported (golint)
pkg/client/registry.go:20:1:warning: exported function GetGenericClient should have comment or be unexported (golint)
pkg/controller/broadcastjob/broadcastjob_controller.go:169:9:warning: should omit 2nd value from range; this loop is equivalent to `for i := range ...` (golint)
pkg/controller/broadcastjob/broadcastjob_controller.go:590:9:warning: if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary) (golint)
pkg/controller/broadcastjob/broadcastjob_utils.go:30:1:warning: exported function IsJobFinished should have comment or be unexported (golint)
DEBUG: [Jul 11 02:21:06.784] nolint: parsing pkg/controller/statefulset/stateful_inplace_utils.go took 679.874µs
DEBUG: [Jul 11 02:21:06.784] nolint: parsing pkg/controller/statefulset/stateful_pod_control.go for directives
pkg/controller/statefulset/stateful_inplace_utils.go:37:1:warning: comment on exported type InPlaceUpdateSpec should be of the form "InPlaceUpdateSpec ..." (with optional leading article) (golint)
pkg/controller/statefulset/stateful_inplace_utils.go:156:2:warning: var patchJson should be patchJSON (golint)
pkg/controller/statefulset/stateful_inplace_utils.go:162:2:warning: var podJson should be podJSON (golint)
pkg/controller/statefulset/stateful_inplace_utils.go:163:2:warning: var newPodJson should be newPodJSON (golint)
pkg/controller/statefulset/stateful_inplace_utils.go:189:2:warning: var inPlaceUpdateStateJson should be inPlaceUpdateStateJSON (golint)
DEBUG: [Jul 11 02:21:06.786] nolint: parsing pkg/controller/statefulset/stateful_pod_control.go took 2.182271ms
DEBUG: [Jul 11 02:21:06.786] nolint: parsing pkg/controller/statefulset/stateful_pod_control_test.go for directives
pkg/controller/statefulset/stateful_pod_control.go:58:1:warning: exported function NewRealStatefulPodControl should have comment or be unexported (golint)
pkg/controller/statefulset/stateful_pod_control.go:168:11:warning: if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary) (golint)
DEBUG: [Jul 11 02:21:06.788] nolint: parsing pkg/controller/statefulset/stateful_pod_control_test.go took 1.816987ms
DEBUG: [Jul 11 02:21:06.788] nolint: parsing pkg/controller/statefulset/stateful_set_control.go for directives
pkg/controller/statefulset/stateful_pod_control_test.go:378:10:warning: if block ends with a return statement, so drop this else and outdent its block (golint)
DEBUG: [Jul 11 02:21:06.790] [golint.3]: golint linter took 459.639266ms
DEBUG: [Jul 11 02:21:06.790] nolint: parsing pkg/controller/statefulset/stateful_set_control.go took 1.646204ms
DEBUG: [Jul 11 02:21:06.790] nolint: parsing pkg/controller/statefulset/stateful_set_status_updater.go for directives
pkg/controller/statefulset/stateful_set_control.go:34:1:warning: comment on exported type StatefulSetControlInterface should be of the form "StatefulSetControlInterface ..." (with optional leading article) (golint)
pkg/controller/statefulset/stateful_set_control.go:36:6:warning: type name will be used as statefulset.StatefulSetControlInterface by other packages, and that stutters; consider calling this ControlInterface (golint)
DEBUG: [Jul 11 02:21:06.790] nolint: parsing pkg/controller/statefulset/stateful_set_status_updater.go took 201.953µs
DEBUG: [Jul 11 02:21:06.790] nolint: parsing pkg/controller/statefulset/stateful_set_status_updater_test.go for directives
pkg/controller/statefulset/stateful_set_status_updater.go:32:6:warning: type name will be used as statefulset.StatefulSetStatusUpdaterInterface by other packages, and that stutters; consider calling this StatusUpdaterInterface (golint)
DEBUG: [Jul 11 02:21:06.791] nolint: parsing pkg/controller/statefulset/stateful_set_status_updater_test.go took 464.611µs
DEBUG: [Jul 11 02:21:06.791] nolint: parsing pkg/controller/statefulset/stateful_set_utils.go for directives
pkg/controller/statefulset/stateful_set_status_updater_test.go:98:10:warning: if block ends with a return statement, so drop this else and outdent its block (golint)
DEBUG: [Jul 11 02:21:06.792] nolint: parsing pkg/controller/statefulset/stateful_set_utils.go took 1.229438ms
DEBUG: [Jul 11 02:21:06.792] nolint: parsing pkg/controller/statefulset/statefulset_controller.go for directives
pkg/controller/statefulset/stateful_set_utils.go:44:5:warning: error var updateConflictError should have name of the form errFoo (golint)
DEBUG: [Jul 11 02:21:06.793] nolint: parsing pkg/controller/statefulset/statefulset_controller.go took 818.562µs
DEBUG: [Jul 11 02:21:06.793] nolint: parsing pkg/util/json.go for directives
pkg/controller/statefulset/statefulset_controller.go:123:1:warning: comment on exported type ReconcileStatefulSet should be of the form "ReconcileStatefulSet ..." (with optional leading article) (golint)
pkg/controller/statefulset/statefulset_controller_test.go:693:10:warning: if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary) (golint)
pkg/controller/statefulset/statefulset_controller_test.go:746:10:warning: if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary) (golint)
DEBUG: [Jul 11 02:21:06.793] nolint: parsing pkg/util/json.go took 53.909µs
DEBUG: [Jul 11 02:21:06.793] nolint: parsing pkg/util/patch/patch.go for directives
pkg/util/json.go:21:1:warning: exported function DumpJson should have comment or be unexported (golint)
pkg/util/json.go:21:6:warning: func DumpJson should be DumpJSON (golint)
DEBUG: [Jul 11 02:21:06.793] nolint: parsing pkg/util/patch/patch.go took 102.553µs
DEBUG: [Jul 11 02:21:06.793] nolint: parsing pkg/webhook/default_server/statefulset/mutating/statefulset_create_update_handler.go for directives
pkg/util/patch/patch.go:33:6:warning: func name will be used as patch.PatchResponseFromRaw by other packages, and that stutters; consider calling this ResponseFromRaw (golint)
pkg/webhook/default_server/pod/mutating/pod_create_handler.go:47:2:warning: exported var SidecarIgnoredNamespaces should have comment or be unexported (golint)
pkg/webhook/default_server/pod/mutating/pod_create_handler.go:97:10:warning: should omit 2nd value from range; this loop is equivalent to `for i := range ...` (golint)
pkg/webhook/default_server/pod/mutating/pod_create_handler.go:130:1:warning: exported function PodMatchSidecarSet should have comment or be unexported (golint)
DEBUG: [Jul 11 02:21:06.793] nolint: parsing pkg/webhook/default_server/statefulset/mutating/statefulset_create_update_handler.go took 432.196µs
DEBUG: [Jul 11 02:21:06.793] nolint: parsing pkg/webhook/default_server/statefulset/validating/statefulset_validation.go for directives
pkg/webhook/default_server/statefulset/mutating/statefulset_create_update_handler.go:92:1:warning: exported function SetObjectDefaults should have comment or be unexported (golint)
DEBUG: [Jul 11 02:21:06.794] nolint: parsing pkg/webhook/default_server/statefulset/validating/statefulset_validation.go took 732.902µs
DEBUG: [Jul 11 02:21:06.794] nolint: parsing pkg/webhook/default_server/statefulset/validating/statefulset_validation_test.go for directives
pkg/webhook/default_server/statefulset/validating/statefulset_validation.go:152:1:warning: comment on exported function ValidateStatefulSetUpdate should be of the form "ValidateStatefulSetUpdate ..." (golint)
pkg/webhook/default_server/statefulset/validating/statefulset_validation.go:185:2:warning: var oldTempJson should be oldTempJSON (golint)
pkg/webhook/default_server/statefulset/validating/statefulset_validation.go:186:2:warning: var newTempJson should be newTempJSON (golint)
DEBUG: [Jul 11 02:21:06.795] nolint: parsing pkg/webhook/default_server/statefulset/validating/statefulset_validation_test.go took 1.233561ms
DEBUG: [Jul 11 02:21:06.795] nolint: parsing pkg/webhook/default_server/utils/default_utils.go for directives
pkg/webhook/default_server/statefulset/validating/statefulset_validation_test.go:77:6:warning: don't use underscores in Go names; var val_1 should be val1 (golint)
DEBUG: [Jul 11 02:21:06.796] nolint: parsing pkg/webhook/default_server/utils/default_utils.go took 490.619µs
DEBUG: [Jul 11 02:21:06.796] nolint: parsing test/e2e/apps/framework.go for directives
pkg/webhook/default_server/utils/default_utils.go:24:1:warning: exported function SetDefaultPodTemplate should have comment or be unexported (golint)
DEBUG: [Jul 11 02:21:06.796] nolint: parsing test/e2e/apps/framework.go took 50.221µs
DEBUG: [Jul 11 02:21:06.796] nolint: parsing test/e2e/apps/types.go for directives
test/e2e/apps/framework.go:22:1:warning: exported function SIGDescribe should have comment or be unexported (golint)
test/e2e/apps/statefulset.go:26:2:warning: should not use dot imports (golint)
test/e2e/apps/statefulset.go:27:2:warning: should not use dot imports (golint)
DEBUG: [Jul 11 02:21:06.796] nolint: parsing test/e2e/apps/types.go took 36.253µs
DEBUG: [Jul 11 02:21:06.796] nolint: parsing test/e2e/framework/cleanup.go for directives
test/e2e/apps/types.go:25:2:warning: exported var NewNginxImage should have comment or be unexported (golint)
DEBUG: [Jul 11 02:21:06.796] nolint: parsing test/e2e/framework/cleanup.go took 122.032µs
DEBUG: [Jul 11 02:21:06.796] nolint: parsing test/e2e/framework/framework.go for directives
test/e2e/framework/cleanup.go:22:6:warning: exported type CleanupActionHandle should have comment or be unexported (golint)
DEBUG: [Jul 11 02:21:06.797] nolint: parsing test/e2e/framework/framework.go took 766.701µs
DEBUG: [Jul 11 02:21:06.797] nolint: parsing test/e2e/framework/pods.go for directives
test/e2e/framework/framework.go:39:2:warning: should not use dot imports (golint)
test/e2e/framework/framework.go:40:2:warning: should not use dot imports (golint)
test/e2e/framework/framework.go:45:2:warning: comment on exported const DefaultNamespaceDeletionTimeout should be of the form "DefaultNamespaceDeletionTimeout ..." (golint)
test/e2e/framework/framework.go:86:6:warning: exported type TestDataSummary should have comment or be unexported (golint)
test/e2e/framework/framework.go:92:6:warning: exported type FrameworkOptions should have comment or be unexported (golint)
test/e2e/framework/framework.go:92:6:warning: type name will be used as framework.FrameworkOptions by other packages, and that stutters; consider calling this Options (golint)
test/e2e/framework/framework.go:98:1:warning: comment on exported function NewDefaultFramework should be of the form "NewDefaultFramework ..." (golint)
test/e2e/framework/framework.go:108:1:warning: exported function NewFramework should have comment or be unexported (golint)
test/e2e/framework/framework.go:269:1:warning: exported method Framework.CreateNamespace should have comment or be unexported (golint)
test/e2e/framework/framework.go:299:1:warning: comment on exported function KruiseDescribe should be of the form "KruiseDescribe ..." (golint)
test/e2e/framework/framework.go:304:1:warning: comment on exported function ConformanceIt should be of the form "ConformanceIt ..." (golint)
DEBUG: [Jul 11 02:21:06.797] nolint: parsing test/e2e/framework/pods.go took 169.681µs
DEBUG: [Jul 11 02:21:06.797] nolint: parsing test/e2e/framework/provider.go for directives
test/e2e/framework/pods.go:31:1:warning: comment on exported method Framework.PodClient should be of the form "PodClient ..." (golint)
test/e2e/framework/pods.go:41:6:warning: exported type PodClient should have comment or be unexported (golint)
DEBUG: [Jul 11 02:21:06.797] nolint: parsing test/e2e/framework/provider.go took 408.736µs
DEBUG: [Jul 11 02:21:06.797] nolint: parsing test/e2e/framework/pv_util.go for directives
test/e2e/framework/provider.go:31:6:warning: exported type Factory should have comment or be unexported (golint)
test/e2e/framework/provider.go:106:1:warning: exported method NullProvider.FrameworkBeforeEach should have comment or be unexported (golint)
test/e2e/framework/provider.go:107:1:warning: exported method NullProvider.FrameworkAfterEach should have comment or be unexported (golint)
test/e2e/framework/provider.go:109:1:warning: exported method NullProvider.ResizeGroup should have comment or be unexported (golint)
test/e2e/framework/provider.go:112:1:warning: exported method NullProvider.GetGroupNodes should have comment or be unexported (golint)
test/e2e/framework/provider.go:115:1:warning: exported method NullProvider.GroupSize should have comment or be unexported (golint)
test/e2e/framework/provider.go:119:1:warning: exported method NullProvider.CreatePD should have comment or be unexported (golint)
test/e2e/framework/provider.go:122:1:warning: exported method NullProvider.DeletePD should have comment or be unexported (golint)
test/e2e/framework/provider.go:125:1:warning: exported method NullProvider.CreatePVSource should have comment or be unexported (golint)
test/e2e/framework/provider.go:128:1:warning: exported method NullProvider.DeletePVSource should have comment or be unexported (golint)
test/e2e/framework/provider.go:132:1:warning: exported method NullProvider.CleanupServiceResources should have comment or be unexported (golint)
test/e2e/framework/provider.go:135:1:warning: exported method NullProvider.EnsureLoadBalancerResourcesDeleted should have comment or be unexported (golint)
DEBUG: [Jul 11 02:21:06.798] nolint: parsing test/e2e/framework/pv_util.go took 117.325µs
test/e2e/framework/provider.go:138:1:warning: exported method NullProvider.LoadBalancerSrcRanges should have comment or be unexported (golint)
DEBUG: [Jul 11 02:21:06.798] nolint: parsing test/e2e/framework/service_util.go for directives
test/e2e/framework/provider.go:141:1:warning: exported method NullProvider.EnableAndDisableInternalLB should have comment or be unexported (golint)
test/e2e/framework/pv_util.go:36:1:warning: comment on exported function CreatePV should be of the form "CreatePV ..." (golint)
DEBUG: [Jul 11 02:21:06.798] nolint: parsing test/e2e/framework/service_util.go took 108.039µs
DEBUG: [Jul 11 02:21:06.798] nolint: parsing test/e2e/framework/statefulset_utils.go for directives
test/e2e/framework/service_util.go:25:1:warning: exported function CreateServiceSpec should have comment or be unexported (golint)
DEBUG: [Jul 11 02:21:06.801] nolint: parsing test/e2e/framework/statefulset_utils.go took 2.90789ms
DEBUG: [Jul 11 02:21:06.801] nolint: parsing test/e2e/framework/test_context.go for directives
test/e2e/framework/statefulset_utils.go:30:2:warning: should not use dot imports (golint)
test/e2e/framework/statefulset_utils.go:51:2:warning: comment on exported const StatefulSetPoll should be of the form "StatefulSetPoll ..." (golint)
test/e2e/framework/statefulset_utils.go:53:2:warning: comment on exported const StatefulSetTimeout should be of the form "StatefulSetTimeout ..." (golint)
test/e2e/framework/statefulset_utils.go:55:2:warning: comment on exported const StatefulPodTimeout should be of the form "StatefulPodTimeout ..." (golint)
test/e2e/framework/statefulset_utils.go:477:10:warning: if block ends with a return statement, so drop this else and outdent its block (golint)
test/e2e/framework/statefulset_utils.go:494:1:warning: comment on exported method StatefulSetTester.WaitForRunningAndNotReady should be of the form "WaitForRunningAndNotReady ..." (golint)
test/e2e/framework/statefulset_utils.go:514:29:warning: method SetHttpProbe should be SetHTTPProbe (golint)
test/e2e/framework/statefulset_utils.go:519:29:warning: method BreakHttpProbe should be BreakHTTPProbe (golint)
test/e2e/framework/statefulset_utils.go:530:29:warning: method BreakPodHttpProbe should be BreakPodHTTPProbe (golint)
test/e2e/framework/statefulset_utils.go:543:29:warning: method RestoreHttpProbe should be RestoreHTTPProbe (golint)
test/e2e/framework/statefulset_utils.go:554:29:warning: method RestorePodHttpProbe should be RestorePodHTTPProbe (golint)
test/e2e/framework/statefulset_utils.go:664:1:warning: receiver name p should be consistent with previous receiver name s for StatefulSetTester (golint)
test/e2e/framework/statefulset_utils.go:879:1:warning: exported function UpdateStatefulSetWithRetries should have comment or be unexported (golint)
DEBUG: [Jul 11 02:21:06.802] nolint: parsing test/e2e/framework/test_context.go took 1.122169ms
DEBUG: [Jul 11 02:21:06.802] nolint: parsing test/e2e/framework/util.go for directives
test/e2e/framework/test_context.go:172:6:warning: exported type CloudConfig should have comment or be unexported (golint)
test/e2e/framework/test_context.go:173:2:warning: struct field ApiEndpoint should be APIEndpoint (golint)
test/e2e/framework/test_context.go:193:5:warning: exported var TestContext should have comment or be unexported (golint)
test/e2e/framework/test_context.go:195:1:warning: comment on exported function RegisterCommonFlags should be of the form "RegisterCommonFlags ..." (golint)
test/e2e/framework/test_context.go:233:1:warning: comment on exported function RegisterClusterFlags should be of the form "RegisterClusterFlags ..." (golint)
test/e2e/framework/test_context.go:284:1:warning: comment on exported function RegisterNodeFlags should be of the form "RegisterNodeFlags ..." (golint)
DEBUG: [Jul 11 02:21:06.805] nolint: parsing test/e2e/framework/util.go took 3.205219ms
test/e2e/framework/util.go:34:2:warning: should not use dot imports (golint)
test/e2e/framework/util.go:35:2:warning: should not use dot imports (golint)
test/e2e/framework/util.go:61:2:warning: comment on exported const PodListTimeout should be of the form "PodListTimeout ..." (golint)
test/e2e/framework/util.go:63:2:warning: comment on exported const PodStartTimeout should be of the form "PodStartTimeout ..." (golint)
test/e2e/framework/util.go:67:2:warning: comment on exported const PodStartShortTimeout should be of the form "PodStartShortTimeout ..." (golint)
test/e2e/framework/util.go:72:2:warning: comment on exported const PodDeleteTimeout should be of the form "PodDeleteTimeout ..." (golint)
test/e2e/framework/util.go:78:2:warning: comment on exported const NamespaceCleanupTimeout should be of the form "NamespaceCleanupTimeout ..." (golint)
test/e2e/framework/util.go:86:2:warning: comment on exported const ServiceStartTimeout should be of the form "ServiceStartTimeout ..." (golint)
test/e2e/framework/util.go:89:2:warning: comment on exported const Poll should be of the form "Poll ..." (golint)
test/e2e/framework/util.go:95:2:warning: comment on exported const ServiceAccountProvisionTimeout should be of the form "ServiceAccountProvisionTimeout ..." (golint)
test/e2e/framework/util.go:99:2:warning: comment on exported const SingleCallTimeout should be of the form "SingleCallTimeout ..." (golint)
test/e2e/framework/util.go:104:2:warning: comment on exported const NodeReadyInitialTimeout should be of the form "NodeReadyInitialTimeout ..." (golint)
test/e2e/framework/util.go:108:2:warning: comment on exported const PodReadyBeforeTimeout should be of the form "PodReadyBeforeTimeout ..." (golint)
test/e2e/framework/util.go:115:2:warning: exported const ServiceRespondingTimeout should have comment (or a comment on this block) or be unexported (golint)
test/e2e/framework/util.go:118:2:warning: comment on exported const ClaimProvisionTimeout should be of the form "ClaimProvisionTimeout ..." (golint)
test/e2e/framework/util.go:121:2:warning: comment on exported const ClaimProvisionShortTimeout should be of the form "ClaimProvisionShortTimeout ..." (golint)
test/e2e/framework/util.go:125:2:warning: comment on exported const ClaimBindingTimeout should be of the form "ClaimBindingTimeout ..." (golint)
test/e2e/framework/util.go:128:2:warning: comment on exported const ClaimDeletingTimeout should be of the form "ClaimDeletingTimeout ..." (golint)
test/e2e/framework/util.go:131:2:warning: comment on exported const PVReclaimingTimeout should be of the form "PVReclaimingTimeout ..." (golint)
test/e2e/framework/util.go:134:2:warning: comment on exported const PVBindingTimeout should be of the form "PVBindingTimeout ..." (golint)
test/e2e/framework/util.go:137:2:warning: comment on exported const PVDeletingTimeout should be of the form "PVDeletingTimeout ..." (golint)
test/e2e/framework/util.go:140:2:warning: comment on exported const RestartNodeReadyAgainTimeout should be of the form "RestartNodeReadyAgainTimeout ..." (golint)
test/e2e/framework/util.go:144:2:warning: comment on exported const RestartPodReadyAgainTimeout should be of the form "RestartPodReadyAgainTimeout ..." (golint)
test/e2e/framework/util.go:168:2:warning: exported var NotReadyTaintTemplate should have comment or be unexported (golint)
test/e2e/framework/util.go:174:1:warning: comment on exported var RunId should be of the form "RunId ..." (golint)
test/e2e/framework/util.go:175:5:warning: var RunId should be RunID (golint)
test/e2e/framework/util.go:177:6:warning: exported type CreateTestingNSFn should have comment or be unexported (golint)
test/e2e/framework/util.go:467:1:warning: exported function DumpAllNamespaceInfo should have comment or be unexported (golint)
test/e2e/framework/util.go:511:6:warning: exported type EventsLister should have comment or be unexported (golint)
test/e2e/framework/util.go:513:1:warning: exported function DumpEventsInNamespace should have comment or be unexported (golint)
test/e2e/framework/util.go:545:1:warning: comment on exported function AllNodesReady should be of the form "AllNodesReady ..." (golint)
test/e2e/framework/util.go:591:1:warning: exported function RestclientConfig should have comment or be unexported (golint)
test/e2e/framework/util.go:607:6:warning: exported type ClientConfigGetter should have comment or be unexported (golint)
test/e2e/framework/util.go:609:1:warning: exported function LoadConfig should have comment or be unexported (golint)
test/e2e/framework/util.go:614:10:warning: if block ends with a return statement, so drop this else and outdent its block (golint)
test/e2e/framework/util.go:622:1:warning: exported function LoadClientset should have comment or be unexported (golint)
test/e2e/framework/util.go:638:1:warning: exported function Logf should have comment or be unexported (golint)
test/e2e/framework/util.go:642:1:warning: exported function Failf should have comment or be unexported (golint)
test/e2e/framework/util.go:654:1:warning: exported function Skipf should have comment or be unexported (golint)
test/e2e/framework/util.go:660:1:warning: exported function ExpectNoError should have comment or be unexported (golint)
test/e2e/framework/util.go:673:1:warning: exported function ExpectNoErrorWithRetries should have comment or be unexported (golint)
test/e2e/framework/util.go:745:1:warning: exported function DumpDebugInfo should have comment or be unexported (golint)
test/e2e/framework/util.go:756:1:warning: comment on exported function WaitForPodRunningInNamespace should be of the form "WaitForPodRunningInNamespace ..." (golint)
test/e2e/framework/util.go:765:1:warning: comment on exported function WaitForPodNameRunningInNamespace should be of the form "WaitForPodNameRunningInNamespace ..." (golint)
test/e2e/framework/util.go:778:1:warning: exported function WaitTimeoutForPodRunningInNamespace should have comment or be unexported (golint)
test/e2e/framework/util.go:815:1:warning: exported function NewKubectlCommand should have comment or be unexported (golint)
test/e2e/framework/util.go:815:40:warning: exported func NewKubectlCommand returns unexported type *framework.kubectlBuilder, which can be annoying to use (golint)
test/e2e/framework/util.go:877:25:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
test/e2e/framework/util.go:886:11:warning: should omit type int from declaration of var rc; it will be inferred from the right-hand side (golint)
test/e2e/framework/util.go:892:22:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
test/e2e/framework/util.go:898:25:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
test/e2e/framework/util.go:939:1:warning: comment on exported function FilterNodes should be of the form "FilterNodes ..." (golint)
test/e2e/framework/util.go:964:1:warning: exported function IsNodeConditionSetAsExpected should have comment or be unexported (golint)
test/e2e/framework/util.go:968:1:warning: exported function IsNodeConditionSetAsExpectedSilent should have comment or be unexported (golint)
test/e2e/framework/util.go:991:13:warning: if block ends with a return statement, so drop this else and outdent its block (golint)
test/e2e/framework/util.go:1019:11:warning: if block ends with a return statement, so drop this else and outdent its block (golint)
test/e2e/framework/util.go:1035:1:warning: exported function IsNodeConditionUnset should have comment or be unexported (golint)
test/e2e/framework/util.go:1120:1:warning: exported function WaitForPodCondition should have comment or be unexported (golint)
```


**Why is this needed**:
Go code format is one essential part of project. And we can use CI system to make all the developers to follow. It brings lot of benefits to the quality of code.

--- a/docs/tutorial/v1/images/guestbook/main.go
+++ b/docs/tutorial/v1/images/guestbook/main.go
@@ -116,6 +116,7 @@ func ListPushHandler(rw http.ResponseWriter, req *http.Request) {
 	rw.Write(data)
 }
 
+// InfoHandler returns DB info
 func InfoHandler(rw http.ResponseWriter, req *http.Request) {
 	info := ""
 
@@ -133,6 +134,7 @@ func InfoHandler(rw http.ResponseWriter, req *http.Request) {
 	rw.Write([]byte(info + "\n"))
 }
 
+// EnvHandler returns environment info
 func EnvHandler(rw http.ResponseWriter, req *http.Request) {
 	environment := make(map[string]string)
 	for _, item := range os.Environ() {
@@ -150,6 +152,7 @@ func EnvHandler(rw http.ResponseWriter, req *http.Request) {
 	rw.Write(data)
 }
 
+// HelloHandler returns "hello"
 func HelloHandler(rw http.ResponseWriter, req *http.Request) {
 	rw.Write([]byte("Hello from guestbook. " +
 		"Your app is up! (Hostname: " +
@@ -162,11 +165,11 @@ func findRedisURL() string {
 	host := os.Getenv("REDIS_MASTER_SERVICE_HOST")
 	port := os.Getenv("REDIS_MASTER_SERVICE_PORT")
 	password := os.Getenv("REDIS_MASTER_SERVICE_PASSWORD")
-	master_port := os.Getenv("REDIS_MASTER_PORT")
+	masterPort := os.Getenv("REDIS_MASTER_PORT")
 
 	if host != "" && port != "" && password != "" {
 		return password + "@" + host + ":" + port
-	} else if master_port != "" {
+	} else if masterPort != "" {
 		return "redis-master:6379"
 	}
 	return ""

--- a/pkg/apis/apps/v1alpha1/broadcastjob_types.go
+++ b/pkg/apis/apps/v1alpha1/broadcastjob_types.go
@@ -71,6 +71,7 @@ type CompletionPolicy struct {
 	TTLSecondsAfterFinished *int32 `json:"ttlSecondsAfterFinished,omitempty" protobuf:"varint,4,opt,name=ttlSecondsAfterFinished"`
 }
 
+// CompletionPolicyType indicates the type of completion policy
 type CompletionPolicyType string
 
 const (
@@ -121,6 +122,7 @@ type BroadcastJobStatus struct {
 	Desired int32 `json:"desired" protobuf:"varint,7,opt,name=desired"`
 }
 
+// JobConditionType indicates valid conditions type of a job
 type JobConditionType string
 
 // These are valid conditions of a job.

--- a/pkg/apis/apps/v1alpha1/sidecarset_types.go
+++ b/pkg/apis/apps/v1alpha1/sidecarset_types.go
@@ -32,6 +32,7 @@ type SidecarSetSpec struct {
 	Containers []SidecarContainer `json:"containers,omitempty"`
 }
 
+// SidecarContainer defines the container of Sidecar
 type SidecarContainer struct {
 	corev1.Container
 }

--- a/pkg/client/generic_client.go
+++ b/pkg/client/generic_client.go
@@ -6,6 +6,7 @@ import (
 	rest "k8s.io/client-go/rest"
 )
 
+// GenericClientset defines a generic client
 type GenericClientset struct {
 	KubeClient   kubeclientset.Interface
 	KruiseClient kruiseclientset.Interface

--- a/pkg/client/registry.go
+++ b/pkg/client/registry.go
@@ -8,6 +8,7 @@ var (
 	genericClient *GenericClientset
 )
 
+// NewRegistry creates clientset by client-go
 func NewRegistry(mgr manager.Manager) error {
 	var err error
 	genericClient, err = newForConfig(mgr.GetConfig())
@@ -17,6 +18,7 @@ func NewRegistry(mgr manager.Manager) error {
 	return nil
 }
 
+// GetGenericClient returns clientset
 func GetGenericClient() GenericClientset {
 	return *genericClient
 }

--- a/pkg/controller/broadcastjob/broadcastjob_controller.go
+++ b/pkg/controller/broadcastjob/broadcastjob_controller.go
@@ -166,7 +166,7 @@ func (r *ReconcileBroadcastJob) Reconcile(request reconcile.Request) (reconcile.
 
 	// convert pod list to a slice of pointers
 	var pods []*corev1.Pod
-	for i, _ := range podList.Items {
+	for i := range podList.Items {
 		pods = append(pods, &podList.Items[i])
 	}
 
@@ -587,14 +587,15 @@ func (r *ReconcileBroadcastJob) createPods(nodeName, namespace string, template 
 	if err := r.Client.Create(context.TODO(), pod); err != nil {
 		r.recorder.Eventf(object, corev1.EventTypeWarning, kubecontroller.FailedCreatePodReason, "Error creating: %v", err)
 		return err
-	} else {
-		accessor, err := meta.Accessor(object)
-		if err != nil {
-			klog.Errorf("parentObject does not have ObjectMeta, %v", err)
-			return nil
-		}
-		klog.Infof("Controller %v created pod %v", accessor.GetName(), pod.Name)
-		r.recorder.Eventf(object, corev1.EventTypeNormal, kubecontroller.SuccessfulCreatePodReason, "Created pod: %v", pod.Name)
 	}
+
+	accessor, err := meta.Accessor(object)
+	if err != nil {
+		klog.Errorf("parentObject does not have ObjectMeta, %v", err)
+		return nil
+	}
+	klog.Infof("Controller %v created pod %v", accessor.GetName(), pod.Name)
+	r.recorder.Eventf(object, corev1.EventTypeNormal, kubecontroller.SuccessfulCreatePodReason, "Created pod: %v", pod.Name)
+
 	return nil
 }

--- a/pkg/controller/broadcastjob/broadcastjob_utils.go
+++ b/pkg/controller/broadcastjob/broadcastjob_utils.go
@@ -27,6 +27,7 @@ import (
 	"k8s.io/klog"
 )
 
+// IsJobFinished returns true when finishing job
 func IsJobFinished(j *appsv1alpha1.BroadcastJob) bool {
 	if j.Spec.CompletionPolicy.Type == appsv1alpha1.Never {
 		return false

--- a/pkg/controller/statefulset/stateful_inplace_utils.go
+++ b/pkg/controller/statefulset/stateful_inplace_utils.go
@@ -34,7 +34,7 @@ import (
 
 var inPlaceUpdatePatchRexp = regexp.MustCompile("/spec/containers/([0-9]+)/image")
 
-// inPlaceUpdateSpec records the images of containers which need to in-place update.
+// InPlaceUpdateSpec records the images of containers which need to in-place update.
 type InPlaceUpdateSpec struct {
 	revision        string
 	patches         []jsonpatch.JsonPatchOperation

--- a/pkg/controller/statefulset/stateful_pod_control.go
+++ b/pkg/controller/statefulset/stateful_pod_control.go
@@ -55,6 +55,7 @@ type StatefulPodControlInterface interface {
 	DeleteStatefulPod(set *appsv1alpha1.StatefulSet, pod *v1.Pod) error
 }
 
+// NewRealStatefulPodControl returns a new realStatefulPodControl
 func NewRealStatefulPodControl(
 	client clientset.Interface,
 	setLister kruiseappslisters.StatefulSetLister,

--- a/pkg/controller/statefulset/stateful_pod_control_test.go
+++ b/pkg/controller/statefulset/stateful_pod_control_test.go
@@ -375,9 +375,8 @@ func TestStatefulPodControlUpdatePodConflictSuccess(t *testing.T) {
 		if !conflict {
 			conflict = true
 			return true, update.GetObject(), apierrors.NewConflict(action.GetResource().GroupResource(), pod.Name, errors.New("conflict"))
-		} else {
-			return true, update.GetObject(), nil
 		}
+		return true, update.GetObject(), nil
 	})
 	pod.Name = "goo-0"
 	if err := control.UpdateStatefulPod(set, pod); err != nil {

--- a/pkg/controller/statefulset/stateful_set_status_updater_test.go
+++ b/pkg/controller/statefulset/stateful_set_status_updater_test.go
@@ -95,9 +95,8 @@ func TestStatefulSetStatusUpdaterUpdateReplicasConflict(t *testing.T) {
 		if !conflict {
 			conflict = true
 			return true, update.GetObject(), apierrors.NewConflict(action.GetResource().GroupResource(), set.Name, errors.New("Object already exists"))
-		} else {
-			return true, update.GetObject(), nil
 		}
+		return true, update.GetObject(), nil
 	})
 	if err := updater.UpdateStatefulSetStatus(set, &status); err != nil {
 		t.Errorf("UpdateStatefulSetStatus returned an error: %s", err)

--- a/pkg/controller/statefulset/statefulset_controller.go
+++ b/pkg/controller/statefulset/statefulset_controller.go
@@ -120,7 +120,7 @@ func newReconciler(mgr manager.Manager) (reconcile.Reconciler, error) {
 
 var _ reconcile.Reconciler = &ReconcileStatefulSet{}
 
-// StatefulSetController reconciles a StatefulSet object
+// ReconcileStatefulSet reconciles a StatefulSet object
 type ReconcileStatefulSet struct {
 	// client interface
 	kruiseClient kruiseclientset.Interface

--- a/pkg/controller/statefulset/statefulset_controller_test.go
+++ b/pkg/controller/statefulset/statefulset_controller_test.go
@@ -687,12 +687,11 @@ func scaleUpStatefulSetController(set *appsv1alpha1.StatefulSet, ssc *StatefulSe
 		if err := assertMonotonicInvariants(set, spc); err != nil {
 			return err
 		}
-		if obj, _, err := spc.setsIndexer.Get(set); err != nil {
+		obj, _, err := spc.setsIndexer.Get(set)
+		if err != nil {
 			return err
-		} else {
-			set = obj.(*appsv1alpha1.StatefulSet)
 		}
-
+		set = obj.(*appsv1alpha1.StatefulSet)
 	}
 	return assertMonotonicInvariants(set, spc)
 }
@@ -740,11 +739,11 @@ func scaleDownStatefulSetController(set *appsv1alpha1.StatefulSet, ssc *Stateful
 		spc.DeleteStatefulPod(set, pod)
 		ssc.deletePod(pod)
 		fakeWorker(ssc)
-		if obj, _, err := spc.setsIndexer.Get(set); err != nil {
+		obj, _, err := spc.setsIndexer.Get(set)
+		if err != nil {
 			return err
-		} else {
-			set = obj.(*appsv1alpha1.StatefulSet)
 		}
+		set = obj.(*appsv1alpha1.StatefulSet)
 	}
 	return assertMonotonicInvariants(set, spc)
 }

--- a/pkg/util/json.go
+++ b/pkg/util/json.go
@@ -18,6 +18,7 @@ package util
 
 import "encoding/json"
 
+// DumpJson returns the JSON encoding
 func DumpJson(o interface{}) string {
 	j, _ := json.Marshal(o)
 	return string(j)

--- a/pkg/webhook/default_server/pod/mutating/pod_create_handler.go
+++ b/pkg/webhook/default_server/pod/mutating/pod_create_handler.go
@@ -94,7 +94,7 @@ func (h *PodCreateHandler) sidecarsetMutatingPod(ctx context.Context, pod *corev
 
 		sidecarSetGeneration[sidecarSet.Name] = sidecarSet.Generation
 
-		for i, _ := range sidecarSet.Spec.Containers {
+		for i := range sidecarSet.Spec.Containers {
 			sidecarContainer := &sidecarSet.Spec.Containers[i]
 
 			// add env to container
@@ -127,6 +127,7 @@ func (h *PodCreateHandler) sidecarsetMutatingPod(ctx context.Context, pod *corev
 	return nil
 }
 
+// PodMatchSidecarSet determines if pod match Selector of sidecar.
 func PodMatchSidecarSet(pod *corev1.Pod, sidecarSet appsv1alpha1.SidecarSet) (bool, error) {
 	selector, err := metav1.LabelSelectorAsSelector(sidecarSet.Spec.Selector)
 	if err != nil {

--- a/pkg/webhook/default_server/statefulset/mutating/statefulset_create_update_handler.go
+++ b/pkg/webhook/default_server/statefulset/mutating/statefulset_create_update_handler.go
@@ -89,6 +89,7 @@ func (h *StatefulSetCreateUpdateHandler) Handle(ctx context.Context, req types.R
 	return resp
 }
 
+// SetObjectDefaults sets object by default
 func SetObjectDefaults(in *appsv1alpha1.StatefulSet) {
 	setDefaults(in)
 	utils.SetDefaultPodTemplate(&in.Spec.Template.Spec)

--- a/pkg/webhook/default_server/statefulset/validating/statefulset_validation.go
+++ b/pkg/webhook/default_server/statefulset/validating/statefulset_validation.go
@@ -149,7 +149,7 @@ func validateStatefulSet(statefulSet *appsv1alpha1.StatefulSet) field.ErrorList 
 	return allErrs
 }
 
-// validateStatefulSetUpdate tests if required fields in the StatefulSet are set.
+// ValidateStatefulSetUpdate tests if required fields in the StatefulSet are set.
 func ValidateStatefulSetUpdate(statefulSet, oldStatefulSet *appsv1alpha1.StatefulSet) field.ErrorList {
 	allErrs := apivalidation.ValidateObjectMetaUpdate(&statefulSet.ObjectMeta, &oldStatefulSet.ObjectMeta, field.NewPath("metadata"))
 

--- a/pkg/webhook/default_server/statefulset/validating/statefulset_validation_test.go
+++ b/pkg/webhook/default_server/statefulset/validating/statefulset_validation_test.go
@@ -74,7 +74,7 @@ func TestValidateStatefulSet(t *testing.T) {
 	var val1 int32 = 1
 	var val2 int32 = 2
 	var val3 int32 = 3
-	var val_1 int32 = -1
+	var minus1 int32 = -1
 	maxUnavailable1 := intstr.FromInt(1)
 	maxUnavailable120Percent := intstr.FromString("120%")
 	successCases := []appsv1alpha1.StatefulSet{
@@ -191,7 +191,7 @@ func TestValidateStatefulSet(t *testing.T) {
 			ObjectMeta: metav1.ObjectMeta{Name: "abc", Namespace: metav1.NamespaceDefault},
 			Spec: appsv1alpha1.StatefulSetSpec{
 				PodManagementPolicy: apps.OrderedReadyPodManagement,
-				Replicas:            &val_1,
+				Replicas:            &minus1,
 				Selector:            &metav1.LabelSelector{MatchLabels: validLabels},
 				UpdateStrategy:      appsv1alpha1.StatefulSetUpdateStrategy{Type: apps.RollingUpdateStatefulSetStrategyType},
 			},
@@ -364,7 +364,7 @@ func TestValidateStatefulSet(t *testing.T) {
 				UpdateStrategy: appsv1alpha1.StatefulSetUpdateStrategy{Type: apps.RollingUpdateStatefulSetStrategyType,
 					RollingUpdate: func() *appsv1alpha1.RollingUpdateStatefulSetStrategy {
 						return &appsv1alpha1.RollingUpdateStatefulSetStrategy{
-							Partition:       &val_1,
+							Partition:       &minus1,
 							PodUpdatePolicy: appsv1alpha1.RecreatePodUpdateStrategyType,
 							MaxUnavailable:  &maxUnavailable1,
 						}

--- a/pkg/webhook/default_server/utils/default_utils.go
+++ b/pkg/webhook/default_server/utils/default_utils.go
@@ -21,6 +21,7 @@ import (
 	v12 "k8s.io/kubernetes/pkg/apis/core/v1"
 )
 
+// SetDefaultPodTemplate sets default pod template
 func SetDefaultPodTemplate(in *v1.PodSpec) {
 	v12.SetDefaults_PodSpec(in)
 	for i := range in.Volumes {

